### DISCUSSION TITLE
Drag and Drop windows

### DISCUSSION THREAD

**Discussion by alicealysia** [YaLTeR/niri#751] [Ideas]
So we already have the ability to "scroll" through the workspace by using Mod+Middle Click, but I was wondering if we could have another bind that lets you "pluck" a window out of its column, and reposition it, like you can in Paper WM.
Something I really like about paper is that it lets me have a very mouse driven workflow, while still being mostly automatic.

**Comment by YaLTeR** [comment]
Well you're in luck because the interactive move PR #547 is what I've been working on finishing lately, and it'll be ready fairly soon. See also a demo in https://mastodon.online/@YaLTeR/113327593317451843

**Reply by alicealysia** [reply]
Damn, I really did pick the perfect time to finally try Niri!
